package/MDAnalysis/coordinates/PDB.py

```diff
@@ -395,22 +421,20 @@ def _read_frame(self, frame):
                 try:
                     cell_dims = np.array([line[6:15], line[15:24],
                                          line[24:33], line[33:40],
-                                         line[40:47], line[47:54]], 
-                                         dtype=np.float32)  
+                                         line[40:47], line[47:54]],
+                                         dtype=np.float32)
                 except ValueError:
                     warnings.warn("Failed to read CRYST1 record, "
                                   "possibly invalid PDB file, got:\n{}"
                                   "".format(line))
                 else:
-                    if (cell_dims == np.array([1, 1, 1, 90, 90, 90], 
-                        dtype=np.float32)).all():
-                        warnings.warn("1 A^3 CRYST1 record," 
-                                      " this is usually a placeholder in"
-                                      " cryo-em structures. Unit cell"
-                                      " dimensions will not be set.")
+                    if np.allclose(cell_dims, np.array([1.0, 1.0, 1.0, 90.0, 90.0, 90.0])):
+                        warnings.warn("1 A^3 CRYST1 record,"
+                                      " this is usually a placeholder."
+                                      " Unit cell dimensions will not be set.")
```

Comment:
Is it true that "unset" simply means that we default to the reader initialization and that will have to be harmonized by fixing  #2698 ?

Reply:
Yes. In this case they are actually set to `np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0])` so I changed the code to reflect that:

```
                        # FIXME: Dimensions set to zeros.
                        # FIXME: This might change with Issue #2698
                        warnings.warn("1 A^3 CRYST1 record,"
                                      " this is usually a placeholder."
                                      " Unit cell dimensions will be set"
                                      " to zeros.")
```